doc/rfcs/00000-rfc-process.md

```diff
@@ -0,0 +1,150 @@
+interminable, chaotic, and repetitive conversations with no clear resolution and
+no realistic process for resolution. The intention of this proposal is to
+provide a clearer process.
+
+Examples that may have benefited from an RFC process:
+
+* [Implement window.fetch into core][]
+
+## Guide-level explanation
+
+You need to follow this process if you intend to make substantial changes to the
+Node.js runtime. Substantial changes include:
+
+* Semantic or syntactic changes to Node.js that are not bugfixes.
+* Removing stable Node.js features.
+* New modules.
+* Bundling of new executables or removal of existing bundled executables.
+
+Changes that do not require an RFC are changes that do not meet the preceding
+criteria. These might include:
+
+* Rephrasing text or reorganizing documentation.
+* Refactoring code.
+* Bugfixes.
+* Changes that improve performance benchmarks.
+* Additions that only affect core developers and not end users.
+
+If you submit a pull request to implement a new feature without going through
+the RFC process, it might be closed with a polite request to submit an RFC
+first.
+
+## Reference-level explanation
+
+To get a substantial feature added to Node.js, first get the RFC merged into the
+`rfcs` directory as a markdown file. At that point the RFC is _accepted_ and may
+be implemented with the goal of eventual inclusion into Node.js.
+
+* Fork the RFC repo <https://github.com/nodejs/node>
+* Copy `doc/rfcs/00000-template.md` to `doc/rfcs/00000-my-feature.md` where
+  `my-feature` is descriptive. Don't assign an RFC number yet.
+* Fill in the RFC.
+* Submit a pull request. The pull request is where we get review of the
+design from the larger community.
+* Once the pull request is opened, the submitter or a core team collaborator
+  should:
+  * Update the RFC file name to change `00000` to the pull request identifier.
+  * Update `doc/rfcs/README.md` in the pull request to include the RFC in the
+    [Accepted RFCs list][].
+* Build consensus and integrate feedback. RFCs that have broad support are much
+more likely to make progress than those that don't receive any comments.
+
+Eventually, one of the [Node.js core team collaborators][] will either accept
+the RFC by merging the pull request, at which point the RFC is accepted, or
+reject it by closing the pull request.
+
+Once an RFC becomes active then authors may implement it and submit the
+feature as a pull request to the Node.js repository. _Active_ is not a rubber
+stamp, and in particular still does not mean the feature will ultimately be
+merged. It does mean that in principle all the major stakeholders have agreed to
+the feature and are amenable to merging it.
+
+Modifications to active RFCs can be done in subsequent PRs. An RFC that makes it
+through the entire process to implementation is _complete_ and is removed from
+the [Accepted RFCs List][].
+
+## Drawbacks
+
+* This approach means a formal process for substantial features where we do not
```

Comment:
What if we only go through this process if there is no consensus on a feature? I'd hate to have to fill out an RFC form whenever I want to make a consistency API change (like adding `AbortSignal` support to another API)

Reply:
> What if we only go through this process if there is no consensus on a feature? 

I'll move "When to use the RFC process?" to the "Unresolved questions" section.

> I'd hate to have to fill out an RFC form whenever I want to make a consistency API change (like adding `AbortSignal` support to another API)

One RFC does not have to correspond to just one PR. You could do a single RFC for `AbortSignal` support generally and then add it to multiple APIs over time.

Reply:
Is that different from what we've done in collaborator summits for project-wide initiatives in the past?

Reply:
Perhaps not, but it removes the necessity of attendance at a summit to propose something. (Perhaps that's not a good thing?)